backends/cadence/fusion_g3/operators/op_quantize.cpp

```diff
@@ -0,0 +1,797 @@
+/*
+ * Copyright (c) Meta Platforms, Inc. and affiliates.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+#include <executorch/kernels/portable/cpu/util/reduce_util.h>
+#include <executorch/runtime/kernel/kernel_includes.h>
+#include <algorithm>
+#include <cinttypes>
+#include <cmath>
+#include <xa_nnlib_kernels_api.h>
+
+using exec_aten::Scalar;
+using exec_aten::ScalarType;
+using exec_aten::Tensor;
+using torch::executor::Error;
+using torch::executor::KernelRuntimeContext;
+
+/* ScalarType in Executorch do not have support for below data types.
+ * So, creating a placeholder for these data types. Once, ScalarTypes is
+ * updated to have support for below data types, these can be removed and 
+ * operator need to be updated accordingly
+ */
+ enum datatype {
+ Ushort = 20,
+ Bits4u = 21,
+ Bits4 = 22
+ };
+
+/**
+ * For an input tensor, use the scale and zero_point arguments to quantize it.
+ */
+namespace cadence {
+namespace impl {
+namespace FusionG3 {
```

Comment:
namespace needs to be G3.  fixed in https://github.com/pytorch/executorch/pull/7061